### ISSUE TITLE
Purgatos doesn't seem to trigger.

### ISSUE THREAD

**Issue by MIC132** [open-duelyst/duelyst#251]
## Summary

Was watching a friend play, he attacked with Purgatos and there was no option to use it's ability.
(Unrelated, but same match Ooz triggered only once despite being attacked twice in the same turn - is there some list of known issues?)

## Impacted services

- [x] The frontend app
- [ ] The game servers
- [ ] Game builds or automation scripts
- [ ] Tests or Github Actions
- [ ] Infrastructure (Terraform)

## Steps to reproduce

1. Have Purgatos
2. Attack?

## Environment information

- Operating system: Windows
- Browser: Chrome

**Comment by willroberts** [comment]
Hi, I just tried to repro this and I'm not sure it's supposed to pop up and give you a choice. It may be randomly selected, which we can confirm by reading the code.

When my Purgatos attacked in a practice game, it automatically selected "deal 3 damage to the enemy general", which worked correctly.

**Comment by MIC132** [comment]
> Hi, I just tried to repro this and I'm not sure it's supposed to pop up and give you a choice. It may be randomly selected, which we can confirm by reading the code.
> 
> When my Purgatos attacked in a practice game, it automatically selected "deal 3 damage to the enemy general", which worked correctly.

Oh ok, the description sounded like you should get to choose. I never played Duelyst before this so I'm not sure how things are supposed to work. I guess I'll close this.
Btw. could you look at Ooz? The situation was that it was attacked twice (second attack killing it) by two different units, but only one Shadow Creep total spawned. "Whenever this takes damage" doesn't imply to me that it would need to survive (unlike say Oropsisaur). I'm just not sure if I should make another issue, or I just fundamentally don't understand some mechanics.

**Comment by willroberts** [comment]
I just tested Ooz as well and didn't see anything unusual. Its card text is "Whenever this minion takes damage, turn a space occupied by an enemy into Shadow Creep". I saw various enemy-occupied tiles turning into Shadow Creep when Ooz took damage.

In general, the current state of card balance is exactly as it was when the game shut down. Most of the bugs/issues filed here are related to the technical aspects of the codebase itself, like upgrading or refactoring software components.

Since we resolved the initial Purgatos issue, I'll mark this as closed. Feel free to open another issue if needed, or stop by the Discord (linked in the README) for smaller questions.

**Comment by MIC132** [comment]
> I just tested Ooz as well and didn't see anything unusual. Its card text is "Whenever this minion takes damage, turn a space occupied by an enemy into Shadow Creep". I saw various enemy-occupied tiles turning into Shadow Creep when Ooz took damage.

I checked the wiki, and it seems for some reason Ooz can turn the same tile into Shadow Creep again, so that's what must have happened in that game.
